inspirehep/modules/records/receivers.py

```diff
@@ -155,18 +155,26 @@ def add_book_autocomplete(sender, json, *args, **kwargs):
     if 'book' not in json.get('document_type', []):
         return
 
-    authors = force_list(get_value(json, 'authors.full_name'))
-    titles = force_list(get_value(json, 'titles.title'))
+    paths = {
+        'imprints.date',
+        'imprints.publisher',
+        'isbns.value',
+    }
 
-    result = json.get('bookautocomplete', [])
-    result.extend(authors)
-    result.extend(titles)
+    authors = force_list(get_value(json, 'authors.full_name', []))
+    titles = force_list(get_value(json, 'titles.title', []))
```

Comment:
I liked the proposal you had in chat that put everything in `paths`... what was the reason to have some there and some outside of it?